bowwowcare/src/components/Button.js

```diff
@@ -1,9 +1,15 @@
-function Button ({size, bgColor="main-color", textColor="white", borderColor="none", children, onClick}) {
+import styled from "styled-components";
+
+function Button ({size, bgColor, textColor="white", borderColor="none", children, onClick, disabled}) {
     return (
-        <button onClick={onClick} className={`h-12 w-full font-bold rounded-md bg-${bgColor} border border-${borderColor} text-${textColor} text-center`}>
+        <StyledButton disabled={disabled} onClick={onClick} className={`h-12 w-full font-bold rounded-md border border-${borderColor} text-${textColor} text-center`}>
```

Comment:
background color가 항상 primary인 건 아니라서 background color도 변수로 받아야 될 것 같습니다. 저번에 말씀드린 dynamic class name 이슈 때문에 아래와 같이 작성하면 될 것 같은데, styled-components를 사용한 이유가 있을까요?
```suggestion
        <button disabled={disabled} onClick={onClick} className={`h-12 w-full font-bold rounded-md ${bgColorVariant}  border ${borderColorVariant} ${textColorVariant} text-center`}>
```

Reply:
처음에 안일한생각으로 button은 배경색이 primary라고 생각했고 .. 필요한 부분만 컴포넌트화해서 StyledProvider 사용하면 되겠다고 했는데 우리는 tailwind를 쓰고있고 바꾸기엔 무리라는걸 깨닫고 차선변경..